lock-api/src/main/java/com/palantir/lock/v2/TimelockService.java

```diff
@@ -86,14 +87,19 @@ default void tryUnlock(Set<LockToken> tokens) {
 
     // todo(gmaretic): implement
     @DoNotDelegate
-    default StartTransactionWithWatchesResponse startTransactionWithWatches() {
-        return StartTransactionWithWatchesResponse.of(
-                startIdentifiedAtlasDbTransaction(), LockWatchState.of(ImmutableMap.of()));
+    default StartTransactionWithWatchesResponse startTransactionWithWatches(OptionalLong lastKnownState) {
```

Comment:
I don't think TimelockService needs to know about lock watches. The separation we have at the moment is a TimelockService class which requires methods used by our API consumers, and the methods required within txns which are held separately. Given SnapshotTransaction and similar should not need lock watches (we are layering this above that level), we needn't make these changes here.

Reply:
Makes sense to only have this on the RPC client and wire it into TransactionStarter and have something appropriate for commit as well